dock/inner.py

```diff
@@ -144,6 +144,7 @@ def __init__(self, git_url, image, git_dockerfile_path=None,
         :param plugin_files: list of str, load plugins also from these files
         :param parent_registry_insecure: bool, allow connecting to parent registry over plain http
         :param target_registries_insecure: bool, allow connecting to target registries over plain http
+        :param pull_base_image: bool, download base image updates from registry
```

Comment:
since @goldmann's issue #126 is about `inside-build`, I think the argument should also be added there